argon/libs/ftdi/lookup_ftdi.cmake

```diff
@@ -0,0 +1,16 @@
+function(lookup_ftdi)
+    if(FTDI_DRIVER_URL AND FTDI_LIB_DIR)
+        message(STATUS "[ftdi] Using predefined driver source")
+        return()
+    endif()
+
+    if(WIN32)
+        set(FTDI_DRIVER_URL "https://www.ftdichip.com/Drivers/CDM/CDM%20v2.12.28%20WHQL%20Certified.zip" PARENT_SCOPE)
+        if(CMAKE_SIZEOF_VOID_P EQUAL 4)
+            set(FTDI_LIB_DIR i386 PARENT_SCOPE)
+        else()
+            set(FTDI_LIB_DIR amd64 PARENT_SCOPE)
+        endif()
+    endif()
```

Comment:
if not windows - won't it crash on `message(STATUS "[FTDI] URL: ${FTDI_DRIVER_URL}")` in CMakeList?

Reply:
It won't crash on printing message status however it will crash later when trying to build. Added error here